concepticondata/conceptlists/Simko-2020-205.tsv

```diff
@@ -0,0 +1,206 @@
+Simko-2020-205-82	82	овес	oves	oat	common plants	60	OAT
+Simko-2020-205-83	83	просо	proso	millet	common plants	931	MILLET
+Simko-2020-205-84	84	пшеница	pšenica	wheat	common plants	1077	WHEAT
+Simko-2020-205-85	85	ръж	răž	rye	common plants	23	RYE
+Simko-2020-205-86	86	слива	sliva	plum	common plants		
+Simko-2020-205-87	87	смрека	smreka	juniper, spruce	common plants	545	SPRUCE
+Simko-2020-205-88	88	трева	treva	grass	common plants	606	GRASS
+Simko-2020-205-89	89	върба	vărba	willow	common plants	818	WILLOW
+Simko-2020-205-90	90	желъд	želăd	acorn	common plants	968	ACORN
+Simko-2020-205-91	91	жито	žito	grain	common plants	605	GRAIN
+Simko-2020-205-92	92	бягам	bjagam	run	common verbs	1519	RUN
+Simko-2020-205-93	93	бия	bija	beat	common verbs	1665	BEAT
+Simko-2020-205-94	94	боли	boli	hurt	common verbs	2411	HURT (SOMEBODY)
+Simko-2020-205-95	95	съм	săm	be	common verbs	1579	BE
+Simko-2020-205-96	96	чеша	češa	comb	common verbs	2070	COMB (VERB)
+Simko-2020-205-97	97	чуя	čuja	hear	common verbs	1408	HEAR
+Simko-2020-205-98	98	дам	dam	give	common verbs	1447	GIVE
+Simko-2020-205-99	99	държа	dărža	hold	common verbs	1448	HOLD
+Simko-2020-205-100	100	яхам	jaham	ride	common verbs	367	RIDE
+Simko-2020-205-101	101	ща	šta	want	common verbs	1784	WANT
+Simko-2020-205-102	102	ида	ida	go	common verbs	695	GO
+Simko-2020-205-103	103	кълна	kălna	swear	common verbs	1939	SWEAR
+Simko-2020-205-104	104	кърмя	kărmja	nurse, feed	common verbs	2472	SUCKLE (BREASTFEED, NURSE)
+Simko-2020-205-105	105	меля	melja	mill, grind	common verbs	1033	GRIND
+Simko-2020-205-106	106	моля	molja	please, pray	common verbs	24	PRAY
+Simko-2020-205-107	107	мълча	mălča	be silent	common verbs	48	BE SILENT
+Simko-2020-205-108	108	пека	peka	bake	common verbs	183	BAKE
+Simko-2020-205-109	109	сея	seja	sow	common verbs	748	SOW SEEDS
+Simko-2020-205-110	110	треса	tresa	tremble	common verbs	52	SHIVER
+Simko-2020-205-111	111	тъка	tăka	weave	common verbs	133	WEAVE
+Simko-2020-205-112	112	варя	varja	boil	common verbs	2493	BOIL
+Simko-2020-205-113	113	возя	vozja	carry	common verbs	700	CARRY
+Simko-2020-205-114	114	вежа	veža	bind	common verbs	1917	TIE
+Simko-2020-205-115	115	живея	živeja	live	common verbs	1422	BE ALIVE
+Simko-2020-205-116	116	баба	baba	grandmother	kinship terms	1496	GRANDMOTHER
+Simko-2020-205-117	117	брат	brat	brother	kinship terms	1262	BROTHER
+Simko-2020-205-118	118	човек	čovek	human	kinship terms	683	PERSON
+Simko-2020-205-119	119	дядо	djado	grandfather	kinship terms	1383	GRANDFATHER
+Simko-2020-205-120	120	дете	dete	child	kinship terms	1801	CHILD (DESCENDANT)
+Simko-2020-205-121	121	девер	dever	brother-in-law	kinship terms	2314	BROTHER-IN-LAW
+Simko-2020-205-122	122	дъщеря	dăšterja	daughter	kinship terms	1357	DAUGHTER
+Simko-2020-205-123	123	люде	ljude	people	kinship terms	789	PEOPLE
+Simko-2020-205-124	124	мъж	măž	man, husband	kinship terms	2646	HUSBAND OR MALE PERSON
+Simko-2020-205-125	125	невеста	nevesta	bride, daughter-in-law	kinship terms	1057	DAUGHTER-IN-LAW
```

Comment:
I'd unmap because bride and daughter-in-law would need to be mapped to two distinct concepts.